cornedbeef/src/main/java/com/swiftkey/cornedbeef/BubbleCoachMark.java

```diff
@@ -135,9 +141,9 @@ protected void updateView(CoachMarkDimens<Integer> popupDimens, CoachMarkDimens<
         // Optional parameters with default values
         protected boolean showBelowAnchor = false;
         protected float target = 0.5f;
-        
-        public BubbleCoachMarkBuilder(Context context, View anchor, String message) {
-            super(context, anchor, message);
+
+        public BubbleCoachMarkBuilder(Context context, View anchor, String message, int textColor) {
```

Comment:
I felt it should go together with the text message since we don't expose the coachmark's text message either.

Although maybe an easier solution would be getting rid of it and let the client pass the `TextView` or layout resource via the other two constructors.